node/blockAPI/shardBlock.go

```diff
@@ -96,6 +100,8 @@ func (sbp *shardAPIBlockProcessor) convertShardBlockBytesToAPIBlock(hash []byte,
 		miniblocks = append(miniblocks, miniblockAPI)
 	}
 
+	sbp.setStatusESDTTransferTransactionsCrossShard(miniblocks)
```

Comment:
rename to something more generic. Something like `applyStatusFilters`. 
We'll definitely have more cases like this in the future.

Reply:
+1